eventRequirement.js

```diff
@@ -769,6 +804,13 @@ OperandVariable.prototype.toJS = function () {
             op: data.operandValueOrObject.op
         };
     }
+    if (data.operandType == "arrayvalue") {
```

Comment:
if it is the same code like with arrayvalue, then you should not duplicate the code, but reuse the existing if case.

Reply:
done